aiven_mysql_migrate/main.py

```diff
@@ -1,27 +1,49 @@
 # Copyright (c) 2020 Aiven, Helsinki, Finland. https://aiven.io/
 from aiven_mysql_migrate import config
 from aiven_mysql_migrate.migration import MySQLMigrateMethod, MySQLMigration
+from dataclasses import asdict
 
+import json
 import logging
 
 LOGGER = logging.getLogger(__name__)
 
+_LOG_LEVELS = {
+    "debug": logging.DEBUG,
+    "info": logging.INFO,
+    "warning": logging.WARNING,
+    "error": logging.ERROR,
+}
 
-def setup_logging(*, debug=False):
+
+def setup_logging(*, log_level: str = "info"):
     log_format = "%(asctime)s\t%(name)s\t%(levelname)s\t%(message)s"
-    log_level = logging.DEBUG if debug else logging.INFO
+    log_level = _LOG_LEVELS.get(log_level.lower(), logging.INFO)
```

Comment:
Maybe we should write logs as json in case if `--json` option is provided. I think it will be less confusing and could also be re-used. What do you think?

Reply:
Sorry I think I didn't get it - Do you mean to write the logs in a structured-json format or to simply write the output to a file ( .json in this case ? )

Reply:
In a structured json format. I just think it won't be consistent if we use json flag with some non-error log level or if error level is used and output is not in json, how are we going to parse it?

Reply:
Or maybe we just do not write logs at all for `json` and `json-file` options.